builtin/mainmenu/tab_simple_main.lua

```diff
@@ -159,6 +163,17 @@ local function main_button_handler(tabview, fields, name, tabdata)
 
 			gamedata.servername			= menudata.favorites[fav_idx].name
 			gamedata.serverdescription	= menudata.favorites[fav_idx].description
+
```

Comment:
Yes, but proper factorisation will remove 90% of the file, as 90% of the file is duplicate. I want to do a network patch so that it can be merged with the 3/4 patch. Do factorisation on a later change?